hist/hist/src/TF1.cxx

```diff
@@ -530,13 +530,13 @@ TF1::TF1(const char *name, const char *formula, Double_t xmin, Double_t xmax, EA
       fXmax = xmin;
    }
    // Create rep formula (no need to add to gROOT list since we will add the TF1 object)
-
+   const auto formulaLength = strlen(formula);
    // First check if we are making a convolution
-   if (TString(formula, 5) == "CONV(" && formula[strlen(formula) - 1] == ')') {
+   if (TString(formula, 5) == "CONV(" && formula[formulaLength - 1] == ')') {
```

Comment:
Here you also need the `strncmp` trick, because that's the line that triggered the error that started all this.

Reply:
Look up "git interactive rebase". You can use it to fix the existing commits (`git commit --fixup xxxx`), and you can also remove the "Added some constants" "Revert Added some constants" while rebasing.
Once you get the hang of it, you will love it!

Reply:
Ah yes, I forgot that. I will change it as soon as I'm home

Reply:
Ahh thanks for the interactive rebase tip. I was looking for something like that 😀